### PR TITLE
Add appSecurity to EJB remote server-to-server FAT

### PR DESCRIPTION
**Pull request by tkburroughs**
Repeat the EJB remote server-to-server FAT with the
appSecurity feature enabled, with servers that have
the minimal required security configuration.

### CODE REVIEW

**Comment by tkburroughs** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.remote_fat/publish/servers/com.ibm.ws.ejbcontainer.remote.fat.SecureRemoteServer/server.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <serverPolicy.csiv2>
+            <layers>
+                <attributeLayer identityAssertionEnabled="true" trustedIdentities="*" /> 
+                <authenticationLayer establishTrustInClient="Supported" mechanisms="GSSUP"/>
+                <transportLayer sslRef="defaultSSLConfig"/>
+            </layers>
+        </serverPolicy.csiv2>
+        <clientPolicy.csiv2>
+            <layers>
+                <attributeLayer identityAssertionEnabled="true" trustedIdentity="j2ee" trustedPassword="j2ee"/>
```
Comment (brideck) [inline dev/com.ibm.ws.ejbcontainer.remote_fat/publish/servers/com.ibm.ws.ejbcontainer.remote.fat.SecureRemoteServer/server.xml]: Is this doing anything anymore?  I see that you've switched the user to "bob" elsewhere.

**Comment by tkburroughs** [comment]
#build

File: dev/com.ibm.ws.ejbcontainer.remote_fat/publish/servers/com.ibm.ws.ejbcontainer.remote.fat.SecureRemoteServer/server.xml (hunk 1 above)
Comment (tkburroughs) [reply dev/com.ibm.ws.ejbcontainer.remote_fat/publish/servers/com.ibm.ws.ejbcontainer.remote.fat.SecureRemoteServer/server.xml]: @brideck Thanks, that piece of configuration is actually not needed for what is being tested; it has been removed.

**Comment by tkburroughs** [comment]
#run-libby-bot
#build

**Review by brideck — APPROVED** [review]

**Pull request merged**